v2.11.0/language-tr.yml

```diff
@@ -0,0 +1,338 @@
+# language: tr
+
+msg-shop-type-admin-regular: "Admin Marketi"
+msg-shop-type-selling: "Satış Marketi"
+msg-shop-type-buying: "Alış Marketi"
+msg-shop-type-trading: "Takas Marketi"
+msg-shop-type-book: "Kitap Marketi"
+
+msg-shop-type-desc-admin-regular: "Sınırsız stoğu var"
+msg-shop-type-desc-selling: "Oyunculara eşya satar"
+msg-shop-type-desc-buying: "Oyunculardan eşya alır"
+msg-shop-type-desc-trading: "Oyuncularla eşya takas eder"
+msg-shop-type-desc-book: "Kitap kopyalarını satar"
+
+msg-shop-object-type-living: "{type}"
+msg-shop-object-type-sign: "Tabela"
+msg-shop-object-type-npc: "Oyuncu olmayan karakter"
+
+msg-selected-shop-type: "&aSeçilen market türü: &6{type} &7({description})"
+msg-selected-shop-object-type: "&aSeçilen varlık türü: &6{type}"
+
+msg-creation-item-selected: |-
+  &aMarket Oluşturma:
+  &e  Market türünü seçmek için blok olmayan bir yere sağ/sol mouse tıklayınız.
+  &e  Market varlığını seçmek için eğilirken blok olmayan bir yere sağ/sol mouse tıklayınız.
+  &e  Eşyaların depolanacağı yeri seçmek için sandık, varil tarzı bloğa sağ tıklayınız.
+  &e  Sonra yerleştirmek istediğiniz bloğa sağ tıklayarak Marketi yerleştirin.
+
+msg-button-previous-page: "&6<- Önceki sayfa ({prev_page}/{max_page})"
+msg-button-previous-page-lore: []
+msg-button-next-page: "&6Sonraki sayfa ({next_page}/{max_page}) ->"
+msg-button-next-page-lore: []
+msg-button-current-page: "&6Sayfa {page}/{max_page}"
+msg-button-current-page-lore:
+- "Şuanki sayfa/tüm sayfalar"
+
+msg-button-name: "&aMarket ismini değiştir"
+msg-button-name-lore:
+msg-button-container: "&aDepolamayı gör"
+msg-button-container-lore:
+- Marketin kullandığı sandık,
+- varil gibi depolamasını gösterir
+msg-button-delete: "&4Sil"
+msg-button-delete-lore:
+- Marketi kapat ve
+- tamamen kaldır
+msg-button-hire: "&aKirala"
+msg-button-hire-lore:
+- Bu market artık senin olur
+
+msg-button-sign-variant: "&aÇeşiti değiştir"
+msg-button-sign-variant-lore:
+- "Tabelanın kullandığı ağaç türünü değiştirir"

```

Comment:
Have you checked if thesse look okay in-game? Maybe it might make sense to split all these button item lore messages over multiple lines, similar to the default messages, to keep the item info box smaller in width.